internal/page/fixtures/attorney.go

```diff
@@ -384,3 +481,27 @@ func Attorney(
 		return nil
 	}
 }
+
+func createAttorney(ctx context.Context, shareCodeStore ShareCodeStore, attorneyStore AttorneyStore, actorUID actoruid.UID, isReplacement, isTrustCorporation bool, lpaOwnerKey dynamo.LpaOwnerKeyType, email string) (*actor.AttorneyProvidedDetails, error) {
+	shareCode := random.String(16)
+	shareCodeData := actor.ShareCodeData{
+		PK:                    dynamo.ShareKey(dynamo.AttorneyShareKey(shareCode)),
+		SK:                    dynamo.ShareSortKey(dynamo.MetadataKey(shareCode)),
+		ActorUID:              actorUID,
+		IsReplacementAttorney: isReplacement,
+		IsTrustCorporation:    isTrustCorporation,
+		LpaOwnerKey:           lpaOwnerKey,
+	}
+
+	attorneyType := actor.TypeAttorney
+	if isReplacement {
+		attorneyType = actor.TypeReplacementAttorney
+	}
+
+	err := shareCodeStore.Put(ctx, attorneyType, shareCode, shareCodeData)
+	if err != nil {
+		return nil, err
+	}
+
+	return attorneyStore.Create(ctx, shareCodeData, email)
```

Comment:
This is fine, it's the price we pay. I'd rather have the actual code and fixtures a mess. It's only for setting up test data so we shouldn't worry too much about it